src/Tuura/Concept/Circuit/Derived.hs

```diff
@@ -106,7 +106,10 @@ handshake11 a b = handshake a b <> initialise a True <> initialise b True
 
 -- TODO: Restrict the initial state so that a=b=1 is not allowed.
 me :: a -> a -> CircuitConcept a
-me a b = fall a ~> rise b <> fall b ~> rise a
+me a b = fall a ~> rise b <> fall b ~> rise a <> invariantConcept [rise a, rise b]
```

Comment:
This is very confusing: the most likely interpretation of `invariantConcept [rise a, rise b]` is that we always have `a = b = 1`. Clearly, the meaning is the opposite. So, we better use something like `never [rise a, rise b]` here.